src/routes/transactions/transactions.controller.ts

```diff
@@ -109,4 +111,49 @@ export class TransactionsController {
       paginationData,
     );
   }
+
+  @ApiOkResponse({
+    type: QueuedItemPage,
+    description: `
+  This endpoint returns all the transactions that are still
```

Comment:
I've commented on this here: https://github.com/5afe/safe-client-gateway-nest/pull/215#discussion_r1062476876

Reply:
I deleted the description in [d1821d2](https://github.com/5afe/safe-client-gateway-nest/pull/215/commits/d1821d25460a597f844c1773d312a01f0621a0d1). I understood we could leave this description for now and review it later, but yes, maybe it's better to remove it at this point, and document all the endpoints later 🙂